src/odemis/driver/xt_client.py

```diff
@@ -1148,36 +1148,6 @@ def mpp_orientation_info(self):
             self.server._pyroClaimOwnership()
             return self.server.mpp_orientation_info()
 
-    def get_aperture_index(self):
```

Comment:
There are some tests related to aperture_index in xt_client_test.py such as def test_get_aperture_index(self):. That will be removed I think.

Reply:
good catch, I fixed it